create_jenkins_job.py

```diff
@@ -203,6 +204,15 @@ def create_job(os_name, job_name, template_file, additional_dict):
                 'mailer_recipients': DEFAULT_MAIL_RECIPIENTS,
             })
 
+        # configure nightly coverage job for clang compiler on linux
+        if os_name == 'linux':
+            create_job(os_name, 'nightly_' + os_name + '_clang', 'ci_job.xml.em', {
+                'cmake_build_type': 'Debug',
+                'compile_on_clang_default': 'true',
+                'time_trigger_spec': PERIODIC_JOB_SPEC,
+                'mailer_recipients': DEFAULT_MAIL_RECIPIENTS,
+            })
+
```

Comment:
Please move this new block to line 185/186 to be close to the other nightly jobs and not be separate from them by the coverage jobs.

Reply:
Makes sense, moved the block